specs/specTemplate/README.md

```diff
@@ -102,15 +103,15 @@ export class ComponentsDemo extends React.Component<{}, {}>{
 
 ## Style API
 
-### Subcomponents (pseudo-elements)
+#### Subcomponents / Children of the Component (pseudo-elements)
```

Comment:
Hi Bruce,

Slots as not the term to be used here. Actually, our "Accepted Children / Predefined Roles" section describes EXACTLY the thing that the `slot` thing does in WebComponents.

And this section here describes custom pseudo-elements that we expose to CSS to style parts of the component Shadow DOM